ecs-cli/modules/clients/aws/ecs/mock/client.go

```diff
@@ -176,6 +176,28 @@ func (mr *MockECSClientMockRecorder) IsActiveCluster(arg0 interface{}) *gomock.C
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "IsActiveCluster", reflect.TypeOf((*MockECSClient)(nil).IsActiveCluster), arg0)
 }
 
+// ListAccountSettings mocks base method
+func (m *MockECSClient) ListAccountSettings(arg0 *ecs0.ListAccountSettingsInput) (*ecs0.ListAccountSettingsOutput, error) {
+	ret := m.ctrl.Call(m, "ListAccountSettings", arg0)
+	ret0, _ := ret[0].(*ecs0.ListAccountSettingsOutput)
+
+	ret1, _ := ret[1].(error)
+	return ret0, ret1
+}
+
+// ListAccountSettings indicates an expected call of ListAccountSettings
+func (mr *MockECSClientMockRecorder) ListAccountSettings(arg0 interface{}) *gomock.Call {
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "ListAccountSettings", reflect.TypeOf((*MockECSClient)(nil).ListAccountSettings), arg0)
+}
+
+// RegisterTaskDefinition mocks base method
+func (m *MockECSClient) RegisterTaskDefinition(arg0 *ecs0.RegisterTaskDefinitionInput) (*ecs0.TaskDefinition, error) {
```

Comment:
do you know why this was added to the mocks? I don't see changes to this func in the client

Reply:
Oh, that's probably because I screwed up a rebase at some point. I'll regenerate them.